check_tags_in_requests.py

```diff
@@ -72,7 +74,7 @@ def _check_requests(self, project, package, rev):
                         project = project[len(prefix):]
                         sr = srprefix
                         break
-            requests = osc.core.get_request_list(apiurl, project, package, None, ['new', 'review'], 'submit')
+            requests = get_request_list_with_history(apiurl, project, package, None, ['new', 'review'], 'submit')
```

Comment:
This didn't use `withfullhistory`, so it should behave the same with the compatibility wrapper. Were you able to find the cause of the difference?

Reply:
Yes, the return values are different. I can search the details tomorrow. for now I'd like to get this in to have a green CI to iterate on improving the quality on this.